core-rust/state-manager/src/types.rs

```diff
@@ -88,6 +94,12 @@ impl IsMerklizableHash for SubstateChangeHash {}
 
 define_wrapped_hash!(EventHash);
 
+impl Display for EventHash {
```

Comment:
in the name of boilerplate reduction, would it be easy enough to make this impl a part of `define_wrapped_hash!()`?